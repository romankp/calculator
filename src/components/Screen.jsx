var React = require('react');

//var Screen = React.createClass({

class Screen extends React.Component {
    // render: function() {
    render() {
        return (
            <div className="screen">
                <span className="screen-content">{this.props.screenContent}</span>
            </div>
        )
    }
// });
};

module.exports = Screen;
