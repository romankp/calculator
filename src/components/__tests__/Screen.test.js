import React from 'react';
import Screen from '../Screen.jsx';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
    .create(<Screen/>)
    .toJSON();
    expect(tree).toMatchSnapshot();
});
