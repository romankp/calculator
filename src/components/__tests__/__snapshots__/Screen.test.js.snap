// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="screen"
>
  <span
    className="screen-content"
  />
</div>
`;
